build_projects/shared-build-targets-utils/Utils/Crossgen.cs

```diff
@@ -12,6 +12,11 @@ public class Crossgen
     {
         private string _coreClrVersion;
         private string _crossGenPath;
+        private List<string> _excludedLibraries = new List<string>() 
```

Comment:
static readonly string[] ?  We don't expect anyone to be modifying this list, right?